src/app/organizations/organization-edit/organization-edit.component.ts

```diff
@@ -12,9 +12,21 @@ import { faClose } from '@fortawesome/free-solid-svg-icons';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class OrganizationEditComponent {
-  organization$: Observable<Organization | undefined> = this.store.select(selectOrganization);
-  isNew$ = this.store.select(selectIsNewOrganization);
+  organization$?: Observable<Organization>;
+  isNew$: Observable<boolean> = of(false);
```

Comment:
```suggestion
  isNew$ = of(false);
```
Typescript can infer the type.